RUFAS/biophysical/manure/storage/open_lot.py

```diff
@@ -0,0 +1,226 @@
+from copy import copy
+
+from RUFAS.biophysical.manure.manure_constants import ManureConstants
+from RUFAS.biophysical.manure.storage.solids_storage_calculator import SolidsStorageCalculator
+from RUFAS.biophysical.manure.storage.storage import Storage
+from RUFAS.biophysical.manure.storage.storage_cover import StorageCover
+from RUFAS.current_day_conditions import CurrentDayConditions
+from RUFAS.data_structures.animal_to_manure_connection import ManureStream
+from RUFAS.rufas_time import RufasTime
+from RUFAS.units import MeasurementUnits
+
+AMMONIA_EMISSION_COEFFICIENT_IN_OPEN_LOTS: float = 0.36
+"""
+Ammonia emission coefficient used for calculating nitrogen loss in an open lot (unitless).
+"""
+
+NITROUS_OXIDE_COEFFICIENT_IN_OPEN_LOTS: float = 0.02
+"""
+Nitrous oxide coefficient used for calculating nitrogen loss in an open lot (unitless).
+"""
+
+LEACHING_COEFFICIENT: float = 0.035
+"""Leaching coefficient used in the calculation of nitrogen loss in a compost bedded pack barn (unitless)."""
+
+
+class OpenLot(Storage):
+    def __init__(self, name: str, cover: StorageCover, storage_time_period: int | None, surface_area: float):
+        super().__init__(
+            name=name,
+            is_housing_emissions_calculator=False,
+            cover=cover,
```

Comment:
```suggestion
            cover=cover,
```
I think this makes no actual functional difference because cover type is not utilized in open lot calculations, but we probably want to set this to "no cover" to avoid confusion - open lots are generally not covered (at least not in the way RuFaS considers covers)